pkg/build/builder/util.go

```diff
@@ -150,3 +154,15 @@ func ParseProxyURL(proxy string) (*url.URL, error) {
 
 	return proxyURL, err
 }
+
+// NameForBuildVolume returns a valid pod volume name for the provided build volume name.
+func NameForBuildVolume(objName string) string {
+	// Volume names must be a valid DNS Label - see https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names
+	return naming.GetName(strings.ToLower(objName), buildVolumeSuffix, kvalidation.DNS1123LabelMaxLength)
+}
+
+// PathForBuildVolume returns the path in the builder container where the build volume is mounted.
```

Comment:
same thing here wrt shared functions between builder and openshift-controller-manager

Reply:
It was decided by the powers that be that this code should be duplicated between the repos instead of shared via library-go, and as such, the library-go pull request has been closed.